helpers/database.js

```diff
@@ -103,6 +102,7 @@ function Migrator (pgp, db) {
 	};
 
 	this.applyRuntimeQueryFile = function (waterCb) {
+		var dirname = require('path').basename(__dirname) === 'helpers' ? path.join(__dirname, '..') : __dirname;
```

Comment:
While this solution isnt the prettiest its a better option than using more NODE_ENVs